src/components/tags/Tags.js

```diff
@@ -1,16 +1,37 @@
+import { useEffect, useState } from "react"
 import { useNavigate } from "react-router-dom"
+import { getSortedTags } from "../../managers/tagManager"
+import "./Tags.css"
 
 export const Tags = () => {
   const navigate = useNavigate()
 
+  const [tags, setTags] = useState([])
+
   const handleClick = () => {
     navigate("/new-tag")
   }
 
```

Comment:
Please include error handler if input field is blank

Reply:
Good catch - I completed this change in TagForm.js, and I believe this fixed the error.